legend-engine-xt-serviceStore-pure/src/main/resources/core_servicestore/metamodel/serviceStore.pure

```diff
@@ -122,8 +123,46 @@ Class <<typemodifiers.abstract>> meta::external::store::service::metamodel::Secu
 {
 }
 
+Class meta::external::store::service::metamodel::SimpleHttpSecurityScheme extends meta::external::store::service::metamodel::SecurityScheme
+{
+  scheme : String[1];
```

Comment:
should we have scheme as enum or some constraints ?
looking at swagger spec should we also have `bearerFormat` as optional property?